### DISCUSSION TITLE
Allure is not working with the latest cypress version (12.15.0)

### DISCUSSION THREAD

**Discussion by BarbaraGodayol** [cypress-io/cypress#27112] [Component Testing]
I have a project using allure: "@shelex/cypress-allure-plugin": "^2.40.0". And in cypress.config.ts I have the properties allure: true,
allureResultsPath: "allure-results", inside "env".
At the end of the execution, allure-results folder is created with all the data.
This was working until 20th of june before the new release 12.15.0. Now, the allure-results folder is not created.
Can someone help me to know what is hapenning with this last version?

**Comment by keplertwofeek** [comment]
Hey @BarbaraGodayol, The issue is already reported here https://github.com/cypress-io/cypress/issues/27113.
